LICENSE

```diff
@@ -200,9 +200,14 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 
-   This product bundles various third-party components also under the
-   Apache Software License 2.0.
+--------------------------------------------------------------------------------
```

Comment:
Looks very (overly?) verbose to me - beside a ton of `-` characters, the information is still the same.

Reply:
It adds the license specifically to gradle.